x-pack/plugins/ml/public/application/explorer/explorer_charts/explorer_charts_container_service.js

```diff
@@ -395,35 +418,39 @@ export const anomalyDataChange = function (
     return chartData.find((point) => point.date === time);
   }
 
-  Promise.all(seriesPromises)
-    .then((response) => {
-      // calculate an overall min/max for all series
-      const processedData = response.map(processChartData);
-      const allDataPoints = reduce(
-        processedData,
-        (datapoints, series) => {
-          each(series, (d) => datapoints.push(d));
-          return datapoints;
-        },
-        []
-      );
-      const overallChartLimits = chartLimits(allDataPoints);
-
-      data.seriesToPlot = response.map((d, i) => ({
-        ...seriesConfigs[i],
-        loading: false,
-        chartData: processedData[i],
-        plotEarliest: chartRange.min,
-        plotLatest: chartRange.max,
-        selectedEarliest: selectedEarliestMs,
-        selectedLatest: selectedLatestMs,
-        chartLimits: USE_OVERALL_CHART_LIMITS ? overallChartLimits : chartLimits(processedData[i]),
-      }));
-      explorerService.setCharts({ ...data });
-    })
-    .catch((error) => {
-      console.error(error);
-    });
+  if (!isGeoMap) {
```

Comment:
I hit an error when I try to view different types of jobs (e.g. high-count and geomap), a click in the overall swimlane results in this error:

![image](https://user-images.githubusercontent.com/230104/105845359-ec7c2400-5fda-11eb-8cd5-64654b56dde9.png)

Since this file takes care of getting data for all charts it needs to consider the possibility of different types of charts.

Reply:
Should be fixed by 478a75725eb997b45a1c62becae5c72bb4d9a1c2